llvm/lib/Target/AArch64/AArch64Processors.td

```diff
@@ -877,6 +905,8 @@ def : ProcessorModel<"cortex-x3", NeoverseN2Model, ProcessorFeatures.X3,
                      [TuneX3]>;
 def : ProcessorModel<"cortex-x4", NeoverseN2Model, ProcessorFeatures.X4,
                      [TuneX4]>;
+def : ProcessorModel<"cortex-x925", NeoverseN2Model, ProcessorFeatures.X925,
```

Comment:
Maybe NeoverseV2Model, as the best fit even if it is quite different? (Some of the older cortex-x cores look like they could be using a newer model too, but that is a different issue).